install-swagger-ui.sh

```diff
@@ -0,0 +1,8 @@
+# Copyright 2015, EMC, Inc.
+
+cd ./static
+rm -rf swagger-ui
+curl -L -o swagger-ui.zip https://github.com/zyoung51/swagger-ui/archive/master.zip
+unzip swagger-ui.zip swagger-ui-master/dist* -d swagger-ui
```

Comment:
Also, I've found that unzip usually isn't installed by default on some distros including Ubuntu, maybe gzip is a more friendly choice?